src/howtoread/index.md

```diff
@@ -1,6 +1,24 @@
 # How To Read The Handbook
 
-## Examples
+This handbook is not an extensive guide on how to use and configure common Linux
+software. The purpose of this document is to explain how to install, configure,
+and maintain Void Linux systems, focusing on the differences from other common
```

Comment:
```suggestion
and maintain Void Linux systems, and to explain differences between common Linux distributions and Void.
```

The wording of this seemed a little off.  Maybe explain is the wrong word too. Highlight? Cover? Something like that. I think what I'm trying to get at is that it's not that this guide focuses on the differences, just that there will be differences and this guide documents them.

Reply:
I did feel iffy about the wording here. I will come up with something better. I always appreciate your reviews and this is very helpful :)